.github/workflows/ci.yaml

```diff
@@ -15,6 +15,7 @@ jobs:
         python-version: ["3.10", "3.11", "3.12", "3.13.0"]
         # macos-12 is an intel runner, macos-14 is a arm64 runner
         platform: [ubuntu-latest, windows-latest, macos-12, macos-14]
+        zarr-version: ["zarr>=2,<3", "zarr==3.0.0b0"]
```

Comment:
Sorry if I lost track of this - I think you convinced me earlier in the comments of this PR that we could just install zarr==3 here and not bother testing with zarr v2?

Reply:
It is only there for this test: https://github.com/zarr-developers/numcodecs/pull/524/files#diff-b3efef438b6c87206e5b2229310709c6b066d5c41d3c8748c66b8c9ddb0d5ed8R13-R14
Might not be worth it.